new/src/sections/contact/contact.tsx

```diff
@@ -1,9 +1,64 @@
+import { DISCORD_INVITE_LINK, EMAIL_LINK, INSTAGRAM_LINK, ITCH_IO_LINK } from "../../constants/contact";
+import discordLogo from "../../assets/logos/discord_logo.svg";
+import emailLogo from "../../assets/logos/email_logo.svg";
+import instagramLogo from "../../assets/logos/instagram_logo.svg";
+import itchIoLogo from "../../assets/logos/itch_io_logo.svg";
+import Button from "../../components/button/button";
+
 const Contact = (): JSX.Element => {
   return (
-    <div id="contact">
-      <div style={{ height: "100vh" }}>Contact</div> {/* Replace */}
+    <div className="contact">
```

Comment:
Please keep the `id="contact"`. This is necessary for the internal links to work.